pkg/kubelet/apis/stats/v1alpha1/types.go

```diff
@@ -86,6 +86,12 @@ type PodStats struct {
 	// +patchMergeKey=name
 	// +patchStrategy=merge
 	Containers []ContainerStats `json:"containers" patchStrategy:"merge" patchMergeKey:"name"`
+	// Stats pertaining to CPU resources.
```

Comment:
Make it clear whether this is the aggregate resources used by the pod (including all containers), or just the pod overhead.